v2/apiserver/internal/core/events.go

```diff
@@ -537,33 +537,41 @@ func (e *eventsService) CancelMany(
 		)
 	}
 
-	events, err := e.eventsStore.CancelMany(ctx, selector)
+	eventCh, count, err := e.eventsStore.CancelMany(ctx, selector)
```

Comment:
Non-blocking nit: Can we change `count` to `affectedCount` or something so that what it represents is more obvious?